src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js

```diff
@@ -0,0 +1,16 @@
+// See here for why this is needed:
+// https://stackoverflow.com/questions/39830580/jest-test-fails-typeerror-window-matchmedia-is-not-a-function
+
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
do we need to keep the other workarounds you added before, like this one: https://github.com/uc-cdis/data-portal/blob/8ee6bbf6a78b2634ff3db4f5923b79b0c9ce231d/src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.test.jsx#L7-L17 ?

Reply:
Or maybe you can reuse that one?

Reply:
Interesting: I actually removed the workaround above for Histogram.test.jsx and it still passes. I'm confused now...

Reply:
Per our discussion, this was moved to setupJest.js with global scope.